pg_sample

```diff
@@ -698,8 +714,13 @@ foreach my $table (@tables) {
     my ($count) = $dbh->selectrow_array("SELECT count(*) FROM $sample_table");
     notice "Exporting data from $sample_table ($count)\n";
   }
-  print "COPY $table FROM stdin;\n";
-  $dbh->do(qq{ COPY $sample_table TO STDOUT });
+  my ($schema_name, $table_name) = split('\.', $table, 2);
+  my $cleaned_table_name = substr $table_name, 1, -1;
+  my $cleaned_schema_name = substr $schema_name, 1, -1;
+  my ($q) = "SELECT string_agg(quote_ident(column_name), ',') FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_NAME = '$cleaned_table_name' AND TABLE_SCHEMA = '$cleaned_schema_name'";
```

Comment:
@mla this was needed to explicitly persist the order of columns, which may differ in prod vs dev db.